npm/cypress-schematic/src/schematics/ng-generate/files/__path__/__name@dasherize__.spec.ts.template

```diff
@@ -0,0 +1,4 @@
+describe('<%= classify(name) %>', () => {
+  it('', () => {
```

Comment:
I think maybe providing an example assertion or some kind of comment might be useful here? I'm not sure. It might be cruft. I'm thinking that the other files that Cypress scaffolds out of the box (when you run `cypress open` in a new project for the first time) are littered with comments. This seems fine to me though, because _do they really need anything more than this in a scaffolded new test?_ 

Other than this one food-for-thought, I think that everything here LGTM.